### PR TITLE
Firetask for mac

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: firetask.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'firetask' do
+  version '4.7'
+  sha256 '4b1f6fbdaf441f84df8e27534517dbdb0a92a6815f39be7d6182409b9130b661'
+
+  url 'http://downloads.firetask.com/firetask-mac/Firetask-Mac.dmg'
+  name 'FireTask'
+  homepage 'http://firetask.com'
+  license :closed    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (vitorgalvao) [inline firetask.rb]: The comment needs to be removed.

**Comment by vitorgalvao** [comment]
Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

**Comment by vitorgalvao** [comment]
Done in bc22b40.

**Comment by twexler** [comment]
This seems to be broken: 
```
 Failed to import: caskroom/cask/firetask
undefined method `cask' for Formulary::FormulaNamespace5ca9cf560be6f7d2ca37a3c6325655e6:Module
```

**Comment by vitorgalvao** [comment]
It is. It’ll be fixed shortly.

**Comment by looby** [comment]
I encountered the same error as @twexler earlier today, but it was just resolved by 36115c1.

**Pull request closed without merging**